src/Microsoft.VisualStudio.IntegrationTestService.Shared/IntegrationService.cs

```diff
@@ -50,6 +50,8 @@ public static IntegrationService GetInstanceFromHostProcess(Process hostProcess)
             return (IntegrationService)Activator.GetObject(typeof(IntegrationService), uri);
         }
 
+        public override object InitializeLifetimeService() => null;
```

Comment:
❔ Won't `IntegrationTestServiceCommands._marshalledService` keep this alive for the correct duration anyway?